common/resolver.go

```diff
@@ -127,13 +127,13 @@ func (r *resolverImpl) cachePut(key string, value string) {
 // so names like /./. aren't valid
 
 // PathRegex regex for destination path
-var PathRegex = regexp.MustCompile(`^/[\w.]*[a-zA-Z][\w.]*/[\w.]*[a-zA-Z][\w.]*$`)
+var PathRegex = regexp.MustCompile(`^/[\w.]*[[:alnum:]][\w.]*/[\w.]*[[:alnum:]][\w.]*$`)
```

Comment:
Yes, the canary uses a 'shortened' UUID for the second part of the component .. and that sometimes results in a purely numeric string, which would fail to match the origin regex (that expects at least one non-numeric character).